Library/Formula/uberftp.rb

```diff
@@ -3,23 +3,18 @@
 class Uberftp < Formula
   desc "Interactive GridFTP client"
   homepage 'http://dims.ncsa.illinois.edu/set/uberftp/'
-  url 'https://github.com/JasonAlt/UberFTP/archive/Version_2_7.tar.gz'
-  sha1 'f185e2ed567eca3484ca230e44a6ffdb4ec69792'
+  url 'https://github.com/JasonAlt/UberFTP/archive/Version_2_8.tar.gz'
+  sha256 '8a397d6ef02bb714bb0cbdb259819fc2311f5d36231783cd520d606c97759c2a'
 
   depends_on 'globus-toolkit'
 
   def install
-    # get the flavor
     globus = Formula["globus-toolkit"].opt_prefix
 
-    core = `"#{globus}/sbin/gpt-query" globus_core`
-    flavor = case core
-      when /gcc64dbg/ then "gcc64dbg"
-      when /gcc32dbg/ then "gcc32dbg"
-    end
+    inreplace "configure", "globus_location/include/globus/gcc64dbg", "globus_location/libexec/include"
```

Comment:
```gpt-query``` is not available anymore in globus version 6.0 and there is no differentiation between  gcc64dbg  and gcc32dbg anymore. In addition, have a look at https://github.com/Homebrew/homebrew/pull/22037#discussion_r5982281, please.
I will prepare a patch and send it to the upstream developers, for the time being this patch is fixing the uberftp compilation errors. I will add a comment to the formula file.